### DISCUSSION TITLE
Android project settings

### DISCUSSION THREAD

**Discussion by svbutko** [svbutko/react-native-template-strong#119] [General]
@itsam, let's continue here so that the issue won't be cluttered.

I'm using next tools:
- Node v20.4.0
- JDK 11 (zulu11)

Also try to build from Android Studio directly

**Comment by itsam** [comment]
Hello @svbutko thank you for your interest.
I am going to try firstly Android Studio and then update my node and JDK and let you know

**Comment by itsam** [comment]
So, 
I have updated Android Studio from Dolphin to Android Studio Flamingo | 2022.2.1 Patch 2 which caused also the installation of JDK 11 (Zulu11) and the update of AGP 7.3.1 (used by the old Android Studio) to AGP 7.4.2 and IT WORKED 😄

Still get the error from command line using ```npx react-native run-android``` which probably don't like some kind of settings... somewhere... somehow.. I will dig into that (any ideas are welcome).

Generally I believe the error concerns AGP version (but that's just me)
Thanks for the heads up!!

**Reply by svbutko** [reply]
Glad to help and that everything works 👍
